fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -444,6 +456,12 @@ protected void runRunningJob() throws AlterCancelException {
         tbl.writeLockOrAlterCancelException();
         try {
             Preconditions.checkState(tbl.getState() == OlapTableState.ROLLUP);
+            TabletInvertedIndex invertedIndex = Catalog.getCurrentInvertedIndex();
+            for (List<AgentTask> tasks : failedAgentTasks.values()) {
+                for (AgentTask task : tasks) {
+                    invertedIndex.getReplica(task.getTabletId(), task.getBackendId()).setBad(true);
```

Comment:
This tablet id is new rollup tabeltid  or base tablet id?

Reply:
new rollup tablet id